clients/da-vinci-client/src/main/java/com/linkedin/davinci/blob/ServerBlobFinder.java

```diff
@@ -0,0 +1,54 @@
+package com.linkedin.davinci.blob;
+
+import com.linkedin.venice.exceptions.VeniceException;
+import com.linkedin.venice.helix.HelixCustomizedViewOfflinePushRepository;
+import com.linkedin.venice.meta.Instance;
+import com.linkedin.venice.meta.Partition;
+import com.linkedin.venice.meta.Version;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+public class ServerBlobFinder implements BlobFinder {
```

Comment:
can we rename to `BlobFinderImpl`?

Reply:
I believe that we should keep the name as is since it used to be blob finder in the previous PR (PR 1033) but we added the server part since it this deals with the server aspect